web/public/locales/ko.json

```diff
@@ -113,17 +113,17 @@
     "legend": "범주",
     "faq": "자주 묻는 질문"
   },
-  "wind": "바람",
-  "solar": "태양",
-  "hydro": "전기",
-  "hydro storage": "전기 저장",
-  "battery storage": "배터리 용량",
+  "wind": "풍력",
+  "solar": "태양광",
+  "hydro": "수력",
+  "hydro storage": "양수",
+  "battery storage": "배터리 저장",
   "biomass": "바이오매스",
   "nuclear": "원자력",
-  "geothermal": "지열의",
+  "geothermal": "지열",
   "gas": "가스",
   "coal": "석탄",
-  "oil": "오일",
+  "oil": "석유",
```

Comment:
Now I have only used a online translator to verify some of these and the new one here seems to translate to petroleum? In that case I think we should go with the old one that translates to oil as that is what we are using in other languages and includes more for example when plants burn raw oil.